recommendations/views.py

```diff
@@ -15,52 +21,60 @@ def recommend_view(request):
 
 
 # narrow the possible recommendations by querying based on submitted forms
-# TODO more than just genres (generic problems)
 def narrow_view(request):
-    form = GenreForm(request.POST)  # TODO generic
+    if request.user.is_authenticated is False:
+        form = FORMS[random.randint(0, len(FORMS) - 1)]  # guests get to test
```

Comment:
Not sure if this is an intended feature but if it is I think it should be more obvious (simply by changing the "Get Recommendations" button text) that logged out users/guests simply demo the recommendation form as I had forgotten about this and was a bit confused when trying to get recommendations as a logged out user.

Reply:
good suggestion!